product_revision/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2025 Axelspace
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Product Revision Management",
+    "summary": "Manage and assign revision information for products.",
+    "author": "Axelspace, Odoo Community Association (OCA)",
+    "website": "https://www.axelspace.com",
+    "license": "AGPL-3",
+    "category": "Inventory",
+    "version": "16.0.1.0.0",
+    "depends": ["base", "product", "mail", "stock"],
```

Comment:
`stock` depends on other modules. So, just `stock` as a dependency is enough.
```suggestion
    "depends": ["stock"],
```